apps/core/views.py

```diff
@@ -3,7 +3,8 @@
 from django.shortcuts import render
 
 
-from apps.core.models import Analysis, Pixel, Pixeler, Tag
+from apps.core.models import Analysis, Pixel, Pixeler, PixelSet, Tag
+from apps.data.models import Entry
```

Comment:
You can group the imports in one line (see line 6, which import different models)

Reply:
nevermind, it is correct